Components/AccordionVertical/assets/arrow.svg

```diff
@@ -0,0 +1 @@
+<svg width="13" height="11" viewBox="0 0 13 11" xmlns="http://www.w3.org/2000/svg"><path d="M6.5 10.75L.25.75h12.5" fill-rule="nonzero" fill="#5F5F5F"/></svg>
```

Comment:
How about calling this what it actually is instead of what it looks like?

`arrow.svg` => `toggler-icon.svg` or something.